tools/peptideshaker/searchgui.xml

```diff
@@ -328,15 +94,15 @@
             -temp_folder `pwd`
             -spectrum_files \$cwd
             -output_folder \$cwd/output
-            -id_params SEARCHGUI_IdentificationParameters.par
+            -id_params $par_target_path
 
             -threads "\${GALAXY_SLOTS:-12}"
 
-            #if $advanced_options.searchgui_advanced.searchgui_advanced_selector == 'advanced'
-                -correct_titles "${advanced_options.searchgui_advanced.correct_titles}"
-                $advanced_options.searchgui_advanced.missing_titles
-                -mgf_splitting "${advanced_options.searchgui_advanced.mgf_splitting}"
-                -mgf_spectrum_count "${advanced_options.searchgui_advanced.mgf_spectrum_count}"
+            #if $searchgui_advanced.searchgui_advanced_selector == 'advanced'
```

Comment:
If you are on it, we need to use single quotes all over here instead of double quotes.